parser/directive.go

```diff
@@ -116,7 +115,15 @@ func parseDirective(pos token.Pos, line string) (directive, error) {
 			case "raw":
 				rpc.Raw = true
 			default:
-				if strings.Contains(field, "=") {
+				if strings.HasPrefix(field, "tag:") {
```

Comment:
Should tags actaully be their own directives?

So you can apply a tag to anything?

```go

//encore:api public
//encore:tag my-stuff
func MyAPI() { ... }

//encore:service
//encore:tag my-stuff
type Service {}
```

Feels more consistent and minimises the "directive" getting more and more busy over time.